libs/features/spectrum2-styles/build.mjs

```diff
@@ -68,7 +68,28 @@ const extractAndTransform = (css, prefix) => {
   return customProperties;
 };
 
+const mergeProperties = (baseProps, additionalProps) => {
```

Comment:
This is also defined in `build-main.mjs`, could we import it to avoid duplicates?

Reply:
There are actually a few duplicate functions that I can pull out and import. But considering we're going in the direction of all the s2 tokens living in the main styles.css file anyway, we could probably get rid of this original `build.mjs` that is building `spectrum2-styles.css` and get rid of `spectrum2-styles.css` too.

Reply:
I don't have the context, but if that is (still) the direction - then your suggestion of removing the unnecessary files sounds good 👍